### PR TITLE
fixes #24365 for facebook case study link(issue 24365)

### LINKED ISSUE #24365: Link to the Facebook case study is broken
There is a broken link that should be pointing towards case study from Facebook. It links to facebook.com/unsupportedbrowser instead.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d1612f19-f267-493a-c230-2ca18e4eb50b
* Version Independent ID: d312db0f-6cf1-87ca-09b0-604a66e18a07
* Content: [Facebook, Google, and external provider authentication in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/social/?view=aspnetcore-6.0&tabs=visual-studio)
* Content Source: [aspnetcore/security/authentication/social/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/authentication/social/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by smandia**
@smandia remember to add the following
Fixes #24365
Thanks for doing this.

### CODE REVIEW

File: aspnetcore/security/authentication/social/index.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Enabling users to sign in with their existing credentials:
 * Is convenient for the users.
 * Shifts many of the complexities of managing the sign-in process onto a third party.
 
-For an example of how social logins can drive traffic and customer conversions, see the case study by [Facebook](https://www.facebook.com/unsupportedbrowser).
+For an example of how social logins can drive traffic and customer conversions, see the case study by [Facebook](https://developers.facebook.com/search/?q=case%20studies&notfound=1).
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/social/index.md:24]:
```suggestion
```

**Review by Rick-Anderson — APPROVED** [review]

**Comment by smandia** [comment]
@Rick-Anderson  Just wondering how would this pull request go away . You have approved it but I still see it .

File: aspnetcore/security/authentication/social/index.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Enabling users to sign in with their existing credentials:
 * Is convenient for the users.
 * Shifts many of the complexities of managing the sign-in process onto a third party.
 
-For an example of how social logins can drive traffic and customer conversions, see the case study by [Facebook](https://www.facebook.com/unsupportedbrowser).
+For an example of how social logins can drive traffic and customer conversions, see the case study by [Facebook](https://developers.facebook.com/search/?q=case%20studies&notfound=1
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/social/index.md:24]:
```suggestion
```

**Comment by Rick-Anderson** [comment]
image

And it doesn't show up in https://github.com/dotnet/AspNetCore.Docs/pulls
image

**Pull request merged**